src/AlgebraicGeometry/Surfaces/EllipticSurface/NeighborStep.jl

```diff
@@ -528,7 +528,16 @@ function transform_to_weierstrass(g::MPolyRingElem, x::MPolyRingElem, y::MPolyRi
                                 switch(num)//switch(den)
                             end
                            )
-    return switch(g_trans), new_trans
+    new_inv_trans = MapFromFunc(F, F, f->begin
+                                switch_num = switch(numerator(f))
+                                switch_den = switch(denominator(f))
+                                interm_res = inv_trans(F(switch_num))//inv_trans(F(switch_den))
+                                num = numerator(interm_res)
+                                den = denominator(interm_res)
+                                switch(num)//switch(den)
+                            end
+                           )
+    return switch(g_trans), new_trans, new_inv_trans
```

Comment:
@HechtiDerLachs is the "Towards inclusion of inverses for Weierstrass trafo." commit meant to be on this PR?

Reply:
I cleaned it up. It can stay I think.

Reply:
No, sorry, it can't! 

This is some hacky improvement which was urgently needed for us to fix a gap in our paper. But it's not mature, yet. In the third case below, it even produces wrong results at the moment! Moreover, the current version is breaking and incompatible with the OSCAR book.

The commit landed here by accident and should just be reverted; probably including @simonbrandhorst 's cleanup. I suggest that we will discuss these changes separately and eventually make another PR.

Reply:
Then please clean it up yourself.